akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala

```diff
@@ -72,16 +83,15 @@ private[akka] object FlightRecorderReader {
       raFile = new RandomAccessFile(flightRecorderFile.toFile, "rw")
       channel = raFile.getChannel
       reader = new FlightRecorderReader(channel)
-      println(reader.structure)
-
-      println("--- ALERT ENTRIES")
-      reader.structure.alertLog.logs.foreach(log ⇒ println(log.richEntries.mkString("\n")))
+      val alerts: Seq[FlightRecorderReader#Entry] = reader.structure.alertLog.logs.flatMap(_.richEntries)
+      val hiFreq: Seq[FlightRecorderReader#Entry] = reader.structure.hiFreqLog.logs.flatMap(_.compactEntries)
+      val loFreq: Seq[FlightRecorderReader#Entry] = reader.structure.loFreqLog.logs.flatMap(_.richEntries)
 
-      println("--- HI FREQUENCY ENTRIES")
-      reader.structure.hiFreqLog.logs.foreach(log ⇒ println(log.compactEntries.mkString("\n")))
+      implicit val ordering = Ordering.fromLessThan[FlightRecorderReader#Entry]((a, b) ⇒ a.timeStamp.isBefore(b.timeStamp))
+      val sorted = SortedSet[FlightRecorderReader#Entry](alerts: _*) ++ hiFreq ++ loFreq
```

Comment:
Can probably be replaced with something more performant, didn't want to spend too much time on it so I just used the first thing coming to my mind.